src/components/Header/Header.jsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import './Header.css';
+import { Link } from 'react-router-dom';
+import {BsCart,BsHeart} from "react-icons/bs";
```

Comment:
Suggestion : You can have one file for all the react icons then import it from there .

Reply:
IMO I guess then that file will just have this line and won't be of much use.
```javascript
export {BsCart,BsHeart,SomeOther... } from "react-icons/bs";
```
Would love to have your Point of view on this though.